src/components/Gauge.js

```diff
@@ -17,6 +17,7 @@ const Gauge = React.createClass({
     dataPointRadius: React.PropTypes.number,
     dataPoints: React.PropTypes.array,
     defaultLabelText: React.PropTypes.string,
+    defaultLabelValueColor: React.PropTypes.string,
```

Comment:
I made the distinction because the value & the text beneath it are usually two different colors. For instance, in the image above and in the product designs they want the value, `300` to be a different color than the text beneath it `Total Users`. 

But the name is a little bit strange. What do you think of changing it to be `defaultValueColor`?